pkg/rancher-components/src/components/StringList/StringList.test.ts

```diff
@@ -0,0 +1,72 @@
+import { mount, shallowMount } from '@vue/test-utils';
+import { StringList } from './index';
+
+describe('StringList.vue', () => {
```

Comment:
I would expect some tests related to the UI behaviors, rather than props binding only.
Kudos in any case for adding some tests in a first place, really! 👏